test/fixtures/prefix/src/main.js

```diff
@@ -0,0 +1,14 @@
+import App from 'fusion-core';
+
+export default async function() {
+  const app = new App('element', el => el);
+  app.middleware((ctx, next) => {
+    if (ctx.url === '/' && ctx.path === '/') {
+      ctx.body = 'ROOT REQUEST';
+    } else if (ctx.url === '/test' && ctx.path === '/test') {
```

Comment:
We aren't stripping any route prefixes from `path`, right?

Reply:
It should strip it from any url related props on ctx, since I strip it before sending req/res to koa